tslib.js

```diff
@@ -273,4 +295,6 @@ var __classPrivateFieldSet;
     exporter("__importDefault", __importDefault);
     exporter("__classPrivateFieldGet", __classPrivateFieldGet);
     exporter("__classPrivateFieldSet", __classPrivateFieldSet);
+    exporter("__createBinding", __createBinding);
+    exporter("__setModuleDefault", __setModuleDefault)

```

Comment:
I'm not sure this actually needs to be exposed, since no TypeScript code will emit `tslib_1.__setModuleDefault` (since its only used by `__importStar`.